### ISSUE TITLE
rds-deployment-existing-ad - Existing VNET in its own RG. Deployment fails

### ISSUE THREAD

**Issue by techronaut** [Azure/azure-quickstart-templates#1447]
rds-deployment-existing-ad   https://github.com/Azure/azure-quickstart-templates/tree/master/rds-deployment-existing-ad

Trying to deploy this to an existing VNET fails (new NICs cannot find VNET)- I believe it's looking for the VNET in the same resource group I'm creating through the template. The VNET exists in its own RG  (I know I can move the resources, but would be cleaner to deploy separately).

Thank you.

**Comment by singhkay** [comment]
@MahendraAgrawal Looks like an enhancement is needed to this template.

**Comment by MahendraAgrawal** [comment]
That's correct. Its looking for VNET in same resource group. For current implementation, this is by design. I will ask our team members if we want to support VNET in different resource group or if we can provide documentation to handle that scenario.

**Comment by danofre** [comment]
Creating a new Resource group using an existing vnet schema would be ideal.

**Comment by ggaecke** [comment]
Got an error while deploying: 

"VM has reported a failure when processing extension 'gateway'. Error message: \\\"The DSC Extension failed to execute: Error downloading https://raw.githubusercontent.com/Azure/azure-QuickStart-Templates/master/rds-deployment//Configuration.zip after 29 attempts: The remote name could not be resolved: 'raw.githubusercontent.com'

From the looks of it, looks like there's 2 // before the Configuration.zip
